### ISSUE TITLE
Support for controllers descriptions

### ISSUE THREAD

**Issue by mblataric** [domaindrivendev/Swashbuckle.AspNetCore#747]
Hi, 

I could not find a way to get controller descriptions in swagger? 
This should be supported in swagger, I have seen various examples how to do this with @Api annotation. 
Ideally, <summary> XML tag on controller would be parsed to controller description. 

Thanks, 
Mario

**Comment by cwe1ss** [comment]
I *think* this has been added with https://github.com/domaindrivendev/Swashbuckle.AspNetCore/commit/da987073ff212e1ed60b8d0b35798b05fe4cdfe8

@domaindrivendev I'm also looking forward to this feature. Thanks for adding it!! 🍺 Will there be a release anytime soon?

**Comment by domaindrivendev** [comment]
Should be available as of official release - v2.5.0

NOTE: You'll need to explicitly opt-in. This is because in a Swagger document, there's no such thing as a "controller" - just operations with tags. By default, Swashbuckle uses controller name as the tag but you can change this via the `TagActionsBy` setting. So, this feature only makes sense so long as you're using that default. See below for the opt-in syntax:

```csharp
services.AddSwaggerGen(c =>
{
    ...

    c.IncludeXmlComments(xmlCommentsPath, true);
});
```

**Comment by cwe1ss** [comment]
@domaindrivendev Thank you very much!
